packages/keystone/fields/SymmetricEncryptedText/utils/decryptInfo.js

```diff
@@ -0,0 +1,28 @@
+const SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO = Symbol('SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO')
+
+
+/**
+ * @param item
+ * @param {string} fieldPath
+ * @param {CipherManagerResult} info
+ */
+function saveDecryptInfo (item, fieldPath, info) {
+    const previouslySaved = item[SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO] || {}
+    previouslySaved[fieldPath] = info
+    item[SYMMETRIC_ENCRYPTED_TEXT_DECRYPT_INFO] = previouslySaved
+}
+
+/**
+ * @param item
+ * @param {string} fieldPath
+ * @returns {CipherManagerResult | undefined}
+ */
+function getDecryptInfo (item, fieldPath) {
```

Comment:
Is it working if we will have two encrypted fields in one model?

Reply:
It is outdated. It would work, because data was stores for each encrypted field.

I removed this, because we can't import from @open-condo/keystone/fields and because magic from EncryptedText was removed